src/views/about.njk

```diff
@@ -4,7 +4,8 @@
 <main>
   <header class="main-header">
     <div class="main-title">
-      <h1>About The Federation</h1>
+      <h1>About</h1>
+      <h2>Behind the scene</h2>
```

Comment:
"Behind the scenes" would be more correct - sorry, have to point these small grammar things out ;)